google/ggrc-core
# GGRC-1996 added proper warning message while importing snapshot

# Issue description

*Import for Snapshot object is not available in GGRC.*

# Steps to test the changes

1. Create a program, control, audit

2. Go to Export page

3. Select Snapshots from dropdown "Export Object Type"

4. Select Snapshot object type - control and select mapped to the audit, audit title> Export

5. Make changes in the csv file and import

Actual Result: non-informative warning message is shown while importing snapshot. The system should not allow importing snapshot

Expected Result: Make a proper warning if import snapshot

# Solution description

Added an extra check to handle a snapshot import issue in import_helper.py file
# Sanity checklist




# PR Review checklist
